modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,249 @@
+= AS2 Connector - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Supported Upgrade Paths
+
+This a direct upgrade from the AS2 connector 4.x to 5.0.0.
+
+== Changes in This Release
+
+* The `Async MDN` is now handled as a separate flow. You need to add a new flow with an endpoint for the async MDN.
+
+* Users can reuse their HTTP listener configuration. Doing so means several changes for you and your trading partners:
+
+** Changing ports for an AS2-receive endpoint/port with the new HTTP listener's endpoint/port configuration
+** Making your trading partners aware of the new URL/port configuration that corresponds to the HTTP listener you are reusing
+** Changes to the load-balancer, if used
+
+* Starting with this version of the AS2 connector, a broader set of ciphers are supported: `AES128_CBC`,`AES192_CBC`,`AES256_CBC`.
+Ciphers such as: `AES128`, `AES192`, `AES256` that were used in earlier versions of the AS2 connector now map to this expanded set.
+
+* The namespace has changed from `as2` to `as2-mule4`. You must delete the old namespace before adding the new one.
+
+* The single global configuration is now divided into separate operation/source global configurations for a "receive endpoint" from a "send endpoint" as these are now mutually exclusive.
```

Comment:
fix slash
```suggestion
* The single global configuration is now divided into separate operation or source global configurations for a "receive endpoint" from a "send endpoint," because these are now mutually exclusive.
```

Reply:
Fixed.